test/unit/legacy_unit/socket/client.c

```diff
@@ -1,14 +0,0 @@
-#include <r_socket.h>
-
-int main() {
```

Comment:
all those progrmas are useful examples. could be written as units, but imho its fine to move them to extras